tests/playwright/src/model/pages/volume-details-page.ts

```diff
@@ -36,23 +34,14 @@ export class VolumeDetailsPage extends BasePage {
   constructor(page: Page, name: string) {
     super(page);
     this.volumeName = name;
-    this.labelName = page.getByLabel('name').and(page.getByText('Volume Details'));
-    this.heading = page.getByRole('heading', { name: this.volumeName });
-    this.closeLink = page.getByRole('link', { name: 'Close Details' });
-    this.backToVolumesLink = page.getByRole('link', { name: 'Go back to Volumes' });
-    this.deleteButton = page.getByRole('button', { name: 'Delete Volume' });
-  }
-
-  async activateTab(tabName: string): Promise<this> {
-    const tabItem = this.page.getByRole('link', { name: tabName, exact: true });
-    await tabItem.waitFor({ state: 'visible', timeout: 2000 });
-    await tabItem.click();
-    return this;
+    this.labelName = this.detailsPage.getByLabel('name').and(page.getByText('Volume Details'));
```

Comment:
In every detailspage child class, there is shared labelName. I am thinking that the breadcrumb elements in a header, could be shared as well. Maybe we could even use `navigation` role. Not sure if it allowed since we have another main navigation use, but not in header. so it might work.